umbraco/Umbraco-CMS
# V8: Remove obsolete umb-permission styling

### Prerequisites


### Description

With #5096 `<umb-permission>` was replaced by `<umb-toggle-group>`. However (as mentioned in #5096), the styling from `<umb-permission>` is still widely used throughout the codebase.

This PR removes all styling from `<umb-permission>` and all usages of it.

#### Testing this PR

The `<umb-permission>` styling is mostly used for the small descriptive texts in the overlays presented when performing various actions on culture variant content:

![image](https://user-images.githubusercontent.com/7405322/56519323-0ead5c00-6542-11e9-8431-17b946498d26.png)

...including various messages when applicable (e.g. if a language variant fails to validate):

![image](https://user-images.githubusercontent.com/7405322/56519369-284ea380-6542-11e9-84d5-c0efe492fa15.png)

On top of that the `<umb-permission>` styling is also used when editing languages (notice the dimmed style for the disabled toggle):

![image](https://user-images.githubusercontent.com/7405322/56519519-8da29480-6542-11e9-82b1-270bc72a16ba.png)

When testing this PR, the overlays for the following actions should be tested:

- Publish
- Publish with descendants
- Scheduled publish
- Unpublish
- Publish from listview
- Unpublish from listview
- Send to publish (send for approval)
- Save

And of course language editing should be tested as well 😄
